backend/app/main.py

```diff
@@ -27,11 +27,13 @@ def on_startup():
 @app.get("/idioms/", response_model=list[IdiomSchema])
 def get_idioms(
     db: SessionDep,
-    offset: int = 0,
+    page: int = 1,
     limit: Annotated[int, Query(le=50)] = 50,
     search: Annotated[str, Query()] = "",
 ) -> list[IdiomSchema]:
     search = search.strip()
+    offset = (page - 1) * limit
```

Comment:
I'd rather do this in the front-end to keep the Api more generic. Please let's try to keep changes in the backend to a minimum in the future, only when strictly needed. Let's keep it this time if it helps you in the front-end, I'll rename the limit query parameter to size in a future pr.